### PR TITLE
Adf in key metrics

### PR DESCRIPTION
**Pull request by mnlevy1981**
### Description of changes:

Add `link_to_ADF.ipynb` to the `key_metrics` example. This required a little modification to `generate_adf_config_file.py` to account for the fact that off-line regridding happened in 092 (CESM output on the CAM-SE grid, and ADF needs fields on the FV grid to plot them)

* [x] Have you followed the guidelines in our [Contributor's Guide](https://ncar.github.io/CUPiD/ContributorGuide.html)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you made sure that the [`pre-commit` checks passed (#8 in Adding Notebooks Guide)](https://ncar.github.io/CUPiD/addingnotebookstocollection.html)?
* [x] Have you [hidden the code cells (#8 in Adding Notebooks Guide)](https://ncar.github.io/CUPiD/addingnotebookstocollection.html) in your notebook?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully tested your changes locally?

### CODE REVIEW

File: examples/key_metrics/config.yml [hunk 1]
```diff
@@ -124,6 +124,18 @@ compute_notebooks:
             regridded_output: False # it looks like output is already on f09 grid, didn't need to regrid time-series file
             base_regridded_output: True
             validation_path: '/glade/campaign/cesm/development/cross-wg/diagnostic_framework/nmse_validation/fv0.9x1.25'
+      link_to_ADF:
+        kernel_name: cupid-infrastructure
+        parameter_groups:
+          none:
+            adf_root: ../../examples/key_metrics/ADF_output/
+            case_year_range: "1_100"
```
Comment (TeaganKing) [inline examples/key_metrics/config.yml:132]: This formatting is a bit confusing. Maybe we could use [1,100] instead?

**Comment by TeaganKing** [comment]
This may be something on my end, but when I ran this, everything was computed with no issue but an additional (undesirable) `key_metrics` directory appeared here: `CUPiD/examples/key_metrics/computed_notebooks/key_metrics` and thus I ran into some issues generating the webpage. I'll try starting with a clean version since it's possible something is outdated in my sandbox.

**Comment by mnlevy1981** [comment]
Two more questions to consider:
1. Should we add the ADF comparison table to the `Link to ADF Output` notebook?
2. Should we add `TAUY` to the variable list so that we generate a lat-lon vector plot? This should also add a second plot to the output in the notebook

**Comment by TeaganKing** [comment]
Based on our conversation I think it sounded like these were two of the key items that users will want to look at, so I would think yes to both of those questions.

**Comment by mnlevy1981** [comment]
From 02cb0ce:

CUPiD ADF Table

**Comment by justin-richling** [comment]
Looks good @mnlevy1981! I guess the only improvement would be to work on the number formatting. Thanks for doing this

**Comment by mnlevy1981** [comment]
eff58b9 adds a row to the table and the vector map:

CUPiD ADF Table 2

**Comment by mnlevy1981** [comment]
> Based on our conversation I think it sounded like these were two of the key items that users will want to look at, so I would think yes to both of those questions.

Done!



> Looks good @mnlevy1981! I guess the only improvement would be to work on the number formatting. Thanks for doing this

Good catch! That's probably an option in the `pandas` `df.to_html()` function... I'll look closer at the documentation...

**Comment by mnlevy1981** [comment]
I found the ADF call to `to_html()`, and added `float_format='{:6g}'.format` to my argument list. In 5c20647, it looks like:

CUPiD ADF Table 3

**Review by TeaganKing — APPROVED** [review]
This looks great!

**Review by justin-richling — APPROVED** [review]
This looks good, thanks!

**Pull request merged**